SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -135,6 +134,7 @@ mkdir -p %{buildroot}%{share_kata}
 mkdir -p %{buildroot}%{coco_path}/libexec
 mkdir -p %{buildroot}/etc/systemd/system/containerd.service.d/
 ln -s /usr/bin/cloud-hypervisor               %{buildroot}%{coco_bin}/cloud-hypervisor
```

Comment:
with decoupling the configs, we can most likely split out things that only kata-containers needs vs things that kata-containers-cc needs: Both packages should only install/uninstall whatever components (binaries, configurations) they need.

Reply:
Are we 100% certain that we will only have one CLH on the host and that it has SNP enabled? If so, we can remove one of these. If not, we may want to keep both symlinks here in case we use the non-snp config, which would point to the non-snp CLH.

Reply:
Leaving here discussed offline